test/runtime/cli.go

```diff
@@ -112,4 +112,26 @@ var _ = Describe("RuntimeCLI", func() {
 		})
 	})
 
+	Context("stdout stderr testing", func() {
+
+		It("root command help should print to stderr", func() {
```

Comment:
As a matter of style, ginkgo encourages using `It()` statements in the way you would with english sentences, ie

`It("tests that the root command help output prints to stderr")`